docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -57,6 +57,23 @@ children:
       url: todo-tutorial-geocoding-service.html
       output: 'web, pdf'
 
+  - title: 'TodoList Tutorial'
+    url: todo-list-tutorial.html
```

Comment:
this link currently broken atm. It can't be fixed until a script on loopback.io is updated. I'll wait until everything is finalized (e.g. the package name for the new example) before I submit a PR over there.

Reply:
What's the issue with `loopback.io`?